### ISSUE TITLE
Can Netmaker (server, client, coredns etc work use IPv6 only (no ipv4 configurations)

### ISSUE THREAD

**Issue by bmullan** [gravitl/netmaker#204]
Can Netmaker (server, client, coredns etc work use IPv6 only (no ipv4 configurations)

Or is there anything that has to have an IPv4 address?

thanks

**Comment by afeiszli** [comment]
There is not currently the ability to add ipv6 only. If someone is willing to take this up it should not be too hard to implement but is out of scope for us at the moment and would likely be about a month before we can implement given the current roadmap.
